mission-control-client/src/App.js

```diff
@@ -8,13 +8,13 @@ import Login from './components/auth/Login';
 import Home from "./components/test/Home";
 import DashboardHome from "./components/dashboard/admin-dashboard/DashboardHome";
 import embedAnalytics from './utils/embedAnalytics';
-import Login from './components/auth/Login'
 
 function App() {
-  
+  const location = useLocation();
+
   useEffect(() => {
     embedAnalytics();
-  }, []);
+  }, [location]);
```

Comment:
Reminder that we need to look into this more and make sure that we're taking advantage of full functionality